packages/client/src/components/app/blocks/TableBlock.svelte

```diff
@@ -93,6 +93,9 @@
       const definition = await API.fetchTableDefinition(dataSource?.tableId)
       schema = definition.schema
       primaryDisplay = definition.primaryDisplay
+    } else if (dataSource?.type === "viewV2") {
+      const schemaResponse = await API.viewV2.getSchema(dataSource?.id)
+      schema = schemaResponse.schema
```

Comment:
Unless we are also setting a primary display field (which I assume we will at some point) it would be better to add logic to the generic `fetchDatasourceSchema` to be able to handle viewV2 datasource as that will let the rest of the client library also be able to use it.

Actually regardless of whether we will add a primary display here or not, it would be important to add viewV2 support to `fetchDatasourceSchema` anyway.